lib/game.js

```diff
@@ -4,23 +4,31 @@ var context = canvas.getContext('2d');
 var canvasWidth =  canvas.width;
 var canvasHeight =  canvas.height;
 
+// var bullet = new Bullet({}, boat);
+var Fish = require('./fish');
+var Boat = require('./boat');
+var Bullet = require('./bullet');
+
+var bullets = []
+
+var boat = new Boat({x: canvasWidth/2, y: 25, width: 100, height: 50, velocity: 5});
+
 canvas.addEventListener('keydown', function(key) {
   if (key.which === 37) {
     boat.moveBoatLeft(boat);
   } else if (key.which === 39) {
     boat.moveBoatRight(boat);
   }
-})
+});
 
 canvas.addEventListener('keypress', function(key) {
   if (key.which === 32) {
-    bullet.shoot(canvas);
+    var bullet = new Bullet({}, boat);
```

Comment:
instantiates bullet upon spacebar, then adds it to the bullets array. 
Rather than having two canvas.addEventListeners we can combine this into the first, then possibly export it into a keybinding file